### PR TITLE
Router to ad

### PR DESCRIPTION
**Pull request by RichardDev101**

### CODE REVIEW

File: frontend/src/components/gallary/AdForm.tsx [hunk 1]
```diff
@@ -0,0 +1,153 @@
+import React, {ChangeEvent, useState} from 'react';
+import {LuFileInput} from "react-icons/lu";
+import axios from "axios";
+import {businessCategories} from "../../enum/BusinessCategory";
```
Comment (chris-diesner) [inline frontend/src/components/gallary/AdForm.tsx:4]: remove unused import

File: frontend/src/components/gallary/Home.tsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+            .then((response) => {
+                setAllAds(response.data)
+            }).catch((e) => console.log(e.message))
+    }
+
+    return (
+        <Container>
+            <div className="pb-20 pt-28 pt-24g rid grid-cols-1 sm:grid-cols-2 md:grid-cols-3 lg:grid-cols-4 xl:grid-cols-5 2xl:grid-cols-6 gap-8">
+                <div>
+                    {allAds.map(ad => <AdCard ad={ad}/>)}
```
Comment (chris-diesner) [inline frontend/src/components/gallary/Home.tsx:23]: missing key prop e.g. key={ad.id}

**Review by chris-diesner — CHANGES_REQUESTED** [review]
looks nice and works as expected - anyway please add key prop while u iterate

File: frontend/src/model/Advertisement.ts [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    businessCategories: BusinessCategory,
+    photosID: string,
+    title: string,
+    aboutYourself: string,
+    detailInformationForService: string,
+    averagePrice: number
+    priceCategories: PriceCategory,
+    contacts: ContactDetail,
+    locations:Address,
+    personsID: String
```
Comment (chris-diesner) [inline frontend/src/model/Advertisement.ts:25]: string instead of String

File: frontend/src/components/gallary/AdForm.tsx (hunk 1 above)
Comment (RichardDev101) [reply frontend/src/components/gallary/AdForm.tsx:4]: has been removed

File: frontend/src/components/gallary/Home.tsx (hunk 2 above)
Comment (RichardDev101) [reply frontend/src/components/gallary/Home.tsx:23]: has been added

File: frontend/src/model/Advertisement.ts (hunk 3 above)
Comment (RichardDev101) [reply frontend/src/model/Advertisement.ts:25]: thx, has been changed to "string"

**Review by chris-diesner — APPROVED** [review]

**Review by chris-diesner — APPROVED** [review]

**Pull request merged**